rubycritic.gemspec

```diff
@@ -25,9 +25,9 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "flog", "4.2.1"
   spec.add_runtime_dependency "reek", "2.0.4"
   spec.add_runtime_dependency "parser", ">= 2.2.0", "< 3.0"
+  spec.add_runtime_dependency "rake", "~> 10"
```

Comment:
I wouldn't even catch the load error - developers requiring the `rake_task` should know upfront that `rake` needs to be installed on their end and the `LoadError` itself is enough feedback on the missing dependency.